docs/snapshotting/snapshot-support.md

```diff
@@ -128,6 +128,11 @@ The snapshot functionality is still in developer preview due to the following:
 
 ### Limitations
 
+- Currently on aarch64 platforms only lower 128 bits of any register are
+  saved due to the limitations of `get/set_one_reg` from `kvm-ioctls` crate
+  that Firecracker uses to interact with KVM. This creates an issue with
+  newer aarch64 cpus with support for registers with width greater then
+  128 bits, that can lead to broken uVM snapshots being taken.
```

Comment:
could elaborate on what a broken snapshot means for the users

Reply:
Also what conditions have to be met for a snapshot to be considered `safe`.

Reply:
added a couple lines about 'uVM failures' in such cases.